### ISSUE TITLE
Simpler way of replacing Styleguidist components

### ISSUE THREAD

**Issue by sapegin** [styleguidist/react-styleguidist#483]
Right now [to use](https://react-styleguidist.js.org/docs/cookbook.html) Redux providers, React Intl, Relay etc. you need to use webpack aliases like this:

```js
// styleguide.config.js
const path = require('path');
module.exports = {
  webpackConfig: {
    resolve: {
      alias: {
        'rsg-components/Wrapper': path.join(__dirname, 'lib/styleguide/Wrapper')
      }
    }
  }
};
```

I think we have enough use cases to add a special config option for that. It could look like this:

```js
// styleguide.config.js
const path = require('path');
module.exports = {
  styleguideComponents: {
    Wrapper: path.join(__dirname, 'lib/styleguide/Wrapper')
  }
};
```

* [x] Add a new config option to the schema (scripts/schemas/config.js)
* [x] Generate webpack aliases based on the new option (scripts/make-webpack-config.js)
* [x] Add docs for the config option (docs/Configuration.md)
* [x] Update examples in the docs to use the new option instead of webpack aliases (mostly the cookbook, but better to check other files)

**Comment by stepancar** [comment]
I think ,we don't need config option for that.
Webpack aliases is simple to understand.
And i think that aliases like this (with full path)
``` javascript
 alias: {
       './node_module/react-styleguidist/rsg-components/Wrapper': pathToYourComponent
 }
```
is more simple to understand and more simple for override.
when i see 'rsg-components/Wrapper' i can't understand what is rsg-components wrapper.
It looks like another package

**Comment by sapegin** [comment]
That’s a good reason to have a simple to understand option ;-)

**Comment by stepancar** [comment]
:D

**Comment by sapegin** [comment]
There‘s a PR: https://github.com/styleguidist/react-styleguidist/issues/504 ;-)

**Comment by sapegin** [comment]
Released in [5.5.0](https://github.com/styleguidist/react-styleguidist/releases/tag/v5.5.0).
